src/GovUk.Education.ExploreEducationStatistics.Common/Model/Data/TableBuilderConfiguration.cs

```diff
@@ -1,7 +1,16 @@
+#nullable enable
 namespace GovUk.Education.ExploreEducationStatistics.Common.Model.Data
 {
     public class TableBuilderConfiguration
     {
-        public TableHeaders TableHeaders { get; set; }
+        public TableHeaders TableHeaders { get; set; } = new();
+
+        public TableBuilderConfiguration Clone()
+        {
+            return new TableBuilderConfiguration
+            {
+                TableHeaders = TableHeaders.Clone()
```

Comment:
Can this not use MemberwiseClone as well?

Reply:
Oh I see it does eventually.

Reply:
Ah, yep it doesn't use it here so we can call `Clone` on the `TableHeaders`.